### PR TITLE
Add test case to test orc dictionary encoding with lots of rows for nested types [databricks]

### PR DESCRIPTION
**Pull request by res-life**
closes [#8809 ](https://github.com/NVIDIA/spark-rapids/issues/8809)

Add test case to test orc dictionary encoding with lots of rows for nested types
By default, it's not running because of it's testing large data.
Added it to a [follow on issue #8849](https://github.com/NVIDIA/spark-rapids/issues/8849) to move it to scale test.

Signed-off-by: Chong Gao <res_life@163.com>

### CODE REVIEW

**Comment by res-life** [comment]
build

**Comment by res-life** [comment]
After the premerge is passed, will disable this test case:

```
assume(true, "TODO: move this case to scale test in future")
====>>>
assume(false, "TODO: move this case to scale test in future")
```

**Comment by res-life** [comment]
build

**Comment by res-life** [comment]
build

File: tests/src/test/scala/com/nvidia/spark/rapids/OrcQuerySuite.scala [hunk 1]
```diff
@@ -26,10 +26,10 @@ import org.apache.hadoop.fs.FileUtil.fullyDelete
 import org.apache.hadoop.fs.Path
 import org.apache.orc.{OrcFile, StripeInformation}
 import org.apache.orc.impl.RecordReaderImpl
-
 import org.apache.spark.{SparkConf, SparkContext}
+
```
Comment (revans2) [inline tests/src/test/scala/com/nvidia/spark/rapids/OrcQuerySuite.scala:30]: Please put the spacing back the way it was before. The build is complaining about it.

**Review by revans2 — APPROVED** [review]
Looks good, just need to fix the style compile errors.

**Comment by res-life** [comment]
build

**Review by revans2 — APPROVED** [review]

**Pull request merged**